src/App.tsx

```diff
@@ -1,14 +1,43 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [status, setStatus] = useState<Status>('all');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const filteredTodos = (): Todo[] => {
```

Comment:
```suggestion
  const getFilteredTodos = (): Todo[] => {
```
The function name should contain a verb
Now it sounds like this variable already contains array

Consider accepting required variables as function arguments instead of taking them from the scope
Then, move this function outside of component